client/pool.c

```diff
@@ -677,6 +674,10 @@ static void clear_nonces_hashtable(struct miner_pool_data *miner)
 
 static int share_can_be_accepted(struct miner_pool_data *miner, xdag_hash_t share, uint64_t task_index)
 {
+	if(!miner) {
```

Comment:
Thanks Marco, I rereviewed that logic and find out, that if conn_data->miner is NULL - pool will crash in share_can_be_accepted function and never get calculate_nopaid_shares function.

Reply:
Good point.